build/build.sh

```diff
@@ -48,6 +48,7 @@ pandoc --verbose \
   --include-after-body=build/plugins/link-highlight.html \
   --include-after-body=build/plugins/table-of-contents.html \
   --include-after-body=build/plugins/lightbox.html \
+  --include-after-body=build/plugins/attributes.html \
```

Comment:
Perhaps we want to keep this disabled by default until we have some more real world testing and we're sure the API is stable?